### PR TITLE
{chem} [intel/2021b] WIEN2k 23.2 update

### PR DESCRIPTION
**Pull request by lcniel**
(created using `eb --new-pr`)

Include [easybuild-easyblocks/pull/3035](https://github.com/easybuilders/easybuild-easyblocks/pull/3035)

### CODE REVIEW

**Comment by lcniel** [comment]
Test report by @lcniel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3035
**FAILED**
Failed during parsing of the easyconfigs, so no ecs were built (1 easyconfigs in total)
vera-gpu2 - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, 1 x NVIDIA Tesla V100-SXM2-32GB, 535.104.05, Python 3.6.8
See https://gist.github.com/lcniel/d4a7727bebb0ec9e7e4f763f82d0067c for a full test report.

**Comment by lcniel** [comment]
Test report by @lcniel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3035
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
vera37-2 - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/lcniel/1032d2cc02c2255a026e694cbf3aafa7 for a full test report.

**Comment by lcniel** [comment]
Test report by @lcniel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3035
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
vera37-2 - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/lcniel/35f57e4441d6335f010d0d5229862d17 for a full test report.

**Comment by lcniel** [comment]
Test report by @lcniel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3035
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
vera37-2 - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/lcniel/5632cfd137f0bd969eea992e63a95866 for a full test report.

**Comment by lcniel** [comment]
Test report by @lcniel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3035
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in total)
vera37-2 - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/lcniel/e014e056d1a91d810f225b6a90d0e343 for a full test report.

**Comment by lcniel** [comment]
Test report by @lcniel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3035
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
vera37-2 - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/lcniel/120547b095165bf13cb3b70e45377cb8 for a full test report.

**Comment by lcniel** [comment]
Test report by @lcniel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3035
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera37-2 - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/lcniel/a6adbcf7ca3ace06ca29e0111a2a4661 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 3035"

**Comment by boegel** [comment]
@lcniel Can you add the missing patch description + checksums?

**Comment by lcniel** [comment]
> @lcniel Can you add the missing patch description + checksums?

Yes, I will take care of it on Monday, I just haven't had time this week.

**Comment by lcniel** [comment]
@boegel All tests now pass.

**Comment by akesandgren** [comment]
Test report by @akesandgren
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3035
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/akesandgren/115c38e8ac386252470493117dc2b3a1 for a full test report.

File: easybuild/easyconfigs/w/WIEN2k/WIEN2k-23.2-intel-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    ('Perl', '5.34.0'),
+    ('DFT-D3', '3.2.0'),
+    ('ELPA', '2021.05.001'),
+    ('FFTW', '3.3.10'),
+    ('libxc', '5.1.6'),
+]
+
+osdependencies = [
+    ('glibc-devel', 'libc6-dev'),  # required for libpthread.
+    ('tcsh', '6.24.01')
```
Comment (akesandgren) [inline easybuild/easyconfigs/w/WIEN2k/WIEN2k-23.2-intel-2021b.eb:40]: This is not how osdeps works, the tuple is supposed to be various names of the package from different OS:es like the one above which is REHL vs DEB names.
So drop the 6.24.01 part.

**Review by akesandgren — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/w/WIEN2k/WIEN2k-23.2-intel-2021b.eb (hunk 1 above)
Comment (lcniel) [reply easybuild/easyconfigs/w/WIEN2k/WIEN2k-23.2-intel-2021b.eb]: Done.

**Review by akesandgren — APPROVED** [review]
LGTM

**Pull request merged**